localstack-core/localstack/services/events/event_rule_engine.py

```diff
@@ -147,7 +147,7 @@ def _evaluate_wildcard(condition: str, value: str) -> bool:
         return re.match(re.escape(condition).replace("\\*", ".+") + "$", value)
 
     @staticmethod
-    def _evaluate_numeric_condition(conditions, value) -> bool:
+    def _evaluate_numeric_condition(conditions: list, value) -> bool:
```

Comment:
intentionally skipping the typings for `value` here? Is it `Any` because we check defensively with isinstance?

Reply:
Yes, I was a bit conflicted about putting type hint and defensively checking, I could add `t.Any`, I think it makes a bit more sense than `int | float` which a check right after?

Reply:
yes 👍 

`int | float` would only cover the happy case, but the method is designed to handle `Any` (i.e., user input)